RELEASE_NOTES.md

```diff
@@ -1,7 +1,11 @@
-# Cosmos SDK v0.46.2 Release Notes
+# Cosmos SDK v0.46.3 Release Notes
```

Comment:
I think we can revert this. This won't be the highlight of v0.46.3 :D

Reply:
Yes, but the release notes for this time focus only on dragonberry only (like 0.45.9).

Reply:
then 0.46.3 will need to go to other branch, and once 46.4 will be released the RELEASE NOTES will have to be updated anyway. So I suggest to keep logical subsequent number, rather then keeping it in an "old" (46.2) state.

Reply:
I mean - if you want to do dragonbery release only then you need to fork from 0.46.2 and cp only relevant PRs. Otherwise need to be consistent with changelog and make sure other important updates are included in release notes.